Taskfile.yml

```diff
@@ -27,6 +27,10 @@ tasks:
     cmds:
       - cmd: go test -v -timeout 60s ./...
 
+  test-2e2:
+    cmds:
```

Comment:
`platforms: [linux, darwin]`
bash will not work on windows :'(

Reply:
not sure about this one ^^

Reply:
in the readme/contributing we recommend to run task command on a git bash terminal